docs/docs/getting-started/client.md

```diff
@@ -490,6 +492,10 @@ Every tab/window maintains its own copy of the local session state; the session
 Using low values for `refetchInterval` will increase network traffic and load on authenticated clients and may impact hosting costs and performance.
 :::
 
+#### Base path
+
+If for some reason you're using a custom path, and your application entry point is not the root of the domain "/" but something else, for example "/my-app/" you can use the `basePath` prop to make NextAuth.js aware of that so that all redirects and session handling work as expected.
```

Comment:
```suggestion
If you are using a custom base path, and your application entry point is not at the root of the domain "/" but something else, for example "/my-app/" you can use the `basePath` prop to make NextAuth.js aware of that so that all redirects and session handling work as expected.
```